cmd/agent/app/settings/runtime_setting_dogstatsd_stats.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package settings
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// dsdStatsRuntimeSetting wraps operations to change log level at runtime.
+type dsdStatsRuntimeSetting string
+
+func (s dsdStatsRuntimeSetting) Description() string {
+	return "Enable/disable the dogstatsd debug stats. Possible values: true, false"
+}
+
+func (s dsdStatsRuntimeSetting) Name() string {
+	return string(s)
+}
+
+func (s dsdStatsRuntimeSetting) Get() (interface{}, error) {
+	return common.DSD.DebugMetricsStats, nil
+}
+
+func (s dsdStatsRuntimeSetting) Set(v interface{}) error {
+	var newValue bool
+
+	// to be cautious, take care of both calls with a string (cli) or a bool (programmaticaly)
+	str, ok := v.(string)
```

Comment:
I'm OK to provide it as a helper but for the `settings` package only: in this case testing both the correct type (bool) and string makes sense as the value comes from the CLI. However, in other parts of the code it would not be idiomatic at all. I've pushed a commit !